### ISSUE TITLE
Configuration of save folders is confusing and broken

### ISSUE THREAD

**Issue by ineuw** [linuxmint/cinnamon#11246]
```
 * Cinnamon 5.4.12
 * Standard release
 * Linux Mint Cinnamon 21
 * Graphics:
  Device-1: Intel CoffeeLake-S GT2 [UHD Graphics 630] vendor: ASRock driver: i915 v: kernel bus-ID: 00:02.0 
  Display: x11 server: X.Org v: 1.21.1.3 driver: X: loaded: modesetting unloaded: fbdev,vesa gpu: i915 resolution: 1664x936~59Hz
  OpenGL: renderer: Mesa Intel UHD Graphics 630 (CFL GT2) v: 4.6 Mesa 22.0.5 direct render: Yes 
 * 64 bit 
 ```
### Issue
Changing the default save folder for gnome-screenshot, using the dconf-editor fails.

### Steps to reproduce
Open deconf editor to gnome-screenshot. change the defaults save folder to the desktop  . . . "/home/user/Desktop" 

### Expected behaviour
accept the new path to save the screenshots. . . . I also tried $HOME/Desktop, but didn't work.

**Comment by mtwebster** [comment]
It's a bit confusing, but:

The 'last-save-directory' is used only for interactive screenshots, either by `gnome-screenshot --interactive` or starting from the start menu. The program updates/remembers this location if you change it from the gui during interactive mode (but only interactive mode).

The 'auto-save-directory' key is where non-interactive screenshots go. When it's un-set (like it is by default)  the Pictures folder is used. You probably want to update this one.

**Comment by ineuw** [comment]
Thanks, I am aware of both.  Made the change and so far it's OK. I am closing this issue.

**Comment by ineuw** [comment]
Since I closed this bug, I ran into many other issues, this included, and tried to resolve them with new installs. This is my second and latest install, and the attached image documents the "dconf" setting which works only if the "auto-save-folder" directory is set before the "last-folder-saved" directory registers my manual selection of "/home/ineuw/Desktop".

Updating the "auto-save-folder" later, is not registered and reverts to the /home/ineuw folder.

![221104_221502_dconf_editor_initial](https://user-images.githubusercontent.com/10367016/200099093-f3cba129-e5de-4aff-8ca5-c4cf9d88c049.jpg)

System registered update with the auto-save-folder showing the same

![221104_232721_auto_update_of_last_save_dir](https://user-images.githubusercontent.com/10367016/200099006-79de6b72-17de-4d77-b51e-3f4b90fcb575.jpg)
